docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md

```diff
@@ -95,15 +95,15 @@ with tf.compat.v1.Session() as sess:
 ```
 
 
-## Download the Pre-Trained Large XLNet Model
+## Downloading the Pretrained Large XLNet Model
 
 Download and unzip an archive with the [XLNet-Large, Cased](https://storage.googleapis.com/xlnet/released_models/cased_L-24_H-1024_A-16.zip).
 
-After the archive is unzipped, the directory `cased_L-12_H-1024_A-16` is created and contains the following files:
+After unzipping the archive, the directory *`cased_L-12_H-1024_A-16`* is created and contains the following files:
 
-* TensorFlow checkpoint (`xlnet_model.ckpt`) containing the pre-trained weights (which is actually 3 files)
-* sentence piece model (`spiece.model`) used for (de)tokenization
-* config file (`xlnet_config.json`) which specifies the hyperparameters of the model
+* TensorFlow checkpoint (*`xlnet_model.ckpt`*) containing the pretrained weights (which is actually 3 files)
+* sentence piece model (*`spiece.model`*) used for (de)tokenization
+* config file (*`xlnet_config.json`*) which specifies the hyperparameters of the model
 
 To get pb-file from the archive contents, you need to do the following.
```

Comment:
```suggestion
To get *`pb-file`* from the archive contents, follow the below instructions:
```

Reply:
```suggestion
To get *`pb-file`* from the archive contents, follow the instructions below:
```
The word "below" should succeed the noun.